packages/cashier/src/components/email-verification-empty-state/__tests__/email-verification-empty-state.test.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import EmailVerificationEmptyState from '../email-verification-empty-state';
+import { StoreProvider } from '../../../hooks';
```

Comment:
```suggestion
import { StoreProvider } from '@deriv/hooks';
```
Since you've created the `@deriv/hooks` alias.

Reply:
@nijil-binary Can’t do that, Test files are excluded from TS in the root `tsconfig` file so TS will raise an error if we use aliases in the test files. To fix that, we would need to change the root config, which would affect the whole project.

<img width="1005" alt="image" src="https://user-images.githubusercontent.com/72082844/195694057-a9295f69-c078-446d-b54c-44a61674d348.png">

<img width="1403" alt="image" src="https://user-images.githubusercontent.com/72082844/195693833-94c426d7-8257-4e4f-b7d1-d4f74131ac18.png">

Should I change it? 🤔

Reply:
Ah, got it. Let's not do that.